docs/install.md

```diff
@@ -1,82 +1,83 @@
+        From a system with an Internet connection, save the module dependencies from the PowerShell Gallery by running the following commands in the PowerShell console:
 
-    ```powershell
-    --8<-- "./docs/snippets/save-module-local-windows.ps1"
-    ```
+        ```powershell
+        --8<-- "./docs/snippets/save-module-local-windows.ps1"
+        ```
 
-    Copy the PowerShell Modules [module dependencies](/powershell-module-for-vmware-cloud-foundation-password-management/#module-dependencies) from the Local Machine to air-gapped facing machine by running the following commands:
+        From the system with the Internet connection, copy the module dependencies to a target system by running the following commands in the PowerShell console:
 
-    ```powershell
-    --8<-- "./docs/snippets/copy-module-local-windows.ps1"
-    ```
+        ```powershell
+        --8<-- "./docs/snippets/copy-module-local-windows.ps1"
+        ```
 
-    Import the PowerShell Modules [module dependencies](/powershell-module-for-vmware-cloud-foundation-password-management/#module-dependencies) from the air-gapped machine by running the following commands:
+        On the target system, import the module dependencies by running the following commands in the PowerShell console:
 
-    ```powershell
-    --8<-- "./docs/snippets/import-module.ps1"
-    ```
+        ```powershell
+        --8<-- "./docs/snippets/import-module.ps1"
+        ```
 
-=== ":fontawesome-brands-linux: &nbsp; Linux"
+    === ":fontawesome-brands-linux: &nbsp; Linux"
 
-    Prerequisite for module install on Linux Machine
+        Prerequisite for module installation on Linux.
 
-    ```bash
-    --8<-- "./docs/snippets/pre-req-linux.sh"
-    ```
+        ```bash
+        --8<-- "./docs/snippets/pre-req-linux.sh"
+        ```
 
-    Save Modules [module dependencies](/powershell-module-for-vmware-cloud-foundation-password-management/#module-dependencies) from the PowerShell Gallery on a non air-gapped machine by running the following commands:
+        From a system with an Internet connection, save the module dependencies from the PowerShell Gallery by running the following commands in the PowerShell console:
 
-    ```powershell
-    --8<-- "./docs/snippets/save-module-local-linux.ps1"
-    ```
+        ```powershell
+        --8<-- "./docs/snippets/save-module-local-linux.ps1"
+        ```
 
-    Copy the PowerShell Modules [module dependencies](/powershell-module-for-vmware-cloud-foundation-password-management/#module-dependencies) from the Local Machine to air-gapped facing machine by running the following commands:
+        From the system with an Internet connection, copy the module dependencies to a target system by running the following commands in the PowerShell console:
```

Comment:
```suggestion
        From a system with an Internet connection, copy the module dependencies to a target system by running the following commands in the PowerShell console:
```

Reply:
This was intententially worded this way as a follow up from the first step and it being the same system. :)